packaging/nuget/tools/init.ps1

```diff
@@ -1,46 +1,70 @@
 param($installPath, $toolsPath, $package, $project)
 
-$nserviceBusKeyPath =  "HKCU:SOFTWARE\NServiceBus" 
+$packageVersion = 'Unknown'
+$noticeEvent = 'noticeEvent'
+$jobName = 'analytics'
 
-$platformKeyPath = "HKCU:SOFTWARE\ParticularSoftware"
-$isNewUser = $true
+# cleanup previous runs
+Get-Job | ? { $_.Name  -eq $jobName } | Remove-Job -Force -ErrorAction SilentlyContinue    
+ 
+if ($package) {
+    $packageVersion = $package.Version
+}
 
-$packageVersion = "Unknown"
+# Define script to run in background job
+$jobScriptBlock = { 
+    param($packageVersion, $noticeEvent)
 
-if($package){
-	$packageVersion = $package.Version
-}
+    # Set Tracing on within the Job
+    Set-PSDebug -Trace 2
 
-#Figure out if this is a first time user
-try {
+    # Setup event forwarding to foreground
+    Register-EngineEvent -SourceIdentifier $noticeEvent -Forward 
+    
+    $nserviceBusKeyPath = 'HKCU:SOFTWARE\NServiceBus' 
+    $platformKeyPath = 'HKCU:SOFTWARE\ParticularSoftware'
+    
+    if ((Test-Path $nserviceBusKeyPath) -or (Test-Path $platformKeyPath)) {
+        New-Event -SourceIdentifier $noticeEvent -Sender "analytics" -MessageData "existing"
```

Comment:
Yes it's used, without it the wait-event at the end of the script would have to wait for the timeout